restaurants/grano.js

```diff
@@ -0,0 +1,23 @@
+const cheerio = require('cheerio');
+const he = require('he');
+
+module.exports = {
+	name: 'Piazza del Grano',
+	url: 'http://www.grano.hu/',
+	getMenu: (body, date = new Date()) => {
+		const $ = cheerio.load(body);
+		const day = date.getDay();
+
+		if ([0, 6].indexOf(day) > 0) {
+			return [];
+		}
+
+		const dailyMenu = $('#ajanlat')
+			.find(`div:nth-child(${day}) h2`)
+			.html();
+
+		return dailyMenu
+			.split(', ')
```

Comment:
I'm uncertain about splitting the text by commas. It could very easily happen that the text contains 2 actual menu items, but for grammar reasons, one of them has an extra comma. This would result in returning 3 separate menu items, of which 2 should not have been actually split.

Anyway, as I'm trying to come up with a new way of returning the menu items (which invalidates the scenario I outlined above), let's ignore this for now, and see if it causes any problems.